src/ui/events.js

```diff
@@ -251,6 +269,15 @@ export type MapBoxZoomEvent = {
  * the event is related to loading of a tile.
  * @property {Coordinate} [coord] The coordinate of the tile if the event has a `dataType` of `source` and
  * the event is related to loading of a tile.
+ * @example
+ * // The `sourcedata` event is an example of `MapDataEvent`.
```

Comment:
@katydecorah or @asheemmamoowala do either of you have an opinion on using backticks in code comments? I'm kind of against it since they don't have the same meaning outside of Markdown, but am not sure if there's a better alternative.

Reply:
Agree that it is only useful in markdown where it'll be formatted when rendered to screen.

Reply:
Agree, I'll updated these lines.